### PR TITLE
refactor: Use dsp-ingest for file upload (DEV-2628)

### PR DESCRIPTION
**Pull request by seakayone**
- **add configuration for dsp-ingest server**
- **upload binaries to dsp-ingest instead of Sipi**

Needs https://github.com/dasch-swiss/dsp-js-lib/pull/639 to be merged, released and integrated for updating dsp-api resources correctly. 

For now I was able to test this locally by publish the dsp-js-lib for my machine using our built in [yalc](https://github.com/wclr/yalc) support:
```sh
> cd dsp-js-lib
> npm run yalc-publish
```

And import it into the app:
```sh
> cd dsp-das
> npm run yalc-add-lib
```

### CODE REVIEW

File: libs/vre/shared/app-resource-properties/src/lib/upload-file.service.ts [hunk 1]
```diff
@@ -1,46 +1,60 @@
...
+            observe: 'body' as const,
+            headers: headers,
+        };
+        const url = `${this._acs.dspIngestConfig.url}/projects/${shortcode}/assets/ingest/${file.name}`;
+        return this._http.post<UploadedFileResponse>(url, file, options).pipe(
+            map((res: UploadedFileResponse) => {
+                let baseUrl = `${this._acs.dspIiifConfig.iiifUrl}/${shortcode}/${res.internalFilename}`;
+                return {
+                    internalFilename: res.internalFilename,
+                    thumbnailUrl: `${baseUrl}/full/256,/0/default.jpg`,
```
Comment (irmastnt) [inline libs/vre/shared/app-resource-properties/src/lib/upload-file.service.ts:54]: make sure it is correct, throws exceptions locally:
       GET https://iiif.dev.dasch.swiss/0811/0FJ4CJ0QFDD-tTYHo455xhK.jpx/full/256,/0/default.jpg 404 (Not Found)

File: apps/dsp-app/src/app/workspace/resource/representation/upload/upload.component.ts [hunk 2]
```diff
@@ -92,27 +97,28 @@ export class UploadComponent implements OnInit {
         // show loading indicator only for files > 1MB
         this.isLoading = this.file.size > 1048576;
 
-        formData.append(this.file.name, this.file);
-        this._upload.upload(formData).subscribe(
-          (res: UploadedFileResponse) => {
+        this._store.select(ProjectsSelectors.currentProject).pipe(
+          filter(v => v !== undefined),
```
Comment (irmastnt) [inline apps/dsp-app/src/app/workspace/resource/representation/upload/upload.component.ts:101]: add take(1) here, otherwise subscription will stay active

**Review by irmastnt — CHANGES_REQUESTED** [review]

File: libs/vre/shared/app-resource-properties/src/lib/upload-file.service.ts (hunk 1 above)
Comment (seakayone) [reply libs/vre/shared/app-resource-properties/src/lib/upload-file.service.ts:54]: This PR is not complete yet and also you will have to run Sipi with the changes from https://github.com/dasch-swiss/dsp-api/pull/3250 which is not integrated yet.

File: apps/dsp-app/src/app/workspace/resource/representation/upload/upload.component.ts (hunk 2 above)
Comment (seakayone) [reply apps/dsp-app/src/app/workspace/resource/representation/upload/upload.component.ts:101]: Thanks, see https://github.com/dasch-swiss/dsp-das/pull/1602/commits/b54ab729f12e5014b1ff4c9fe45cedc7d635a51b

**Review by derschnee68 — APPROVED** [review]

**Review by irmastnt — APPROVED** [review]

**Pull request merged**